test/models/address_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class AddressTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'valid state' do
+    address = create(:address)
+    assert address.valid?
+  end
+
+  test 'invalid without name' do
+    address = create(:address)
```

Comment:
these two "invalid without" tests could be combined for simplicity and Address.new will give you an address w/o any fields populated